docs/getting-started/keeping-backstage-updated.md

```diff
@@ -0,0 +1,33 @@
+---
+id: keeping-backstage-updated
+title: Keeping Backstage Updated
+description: How to keep your Backstage App updated
+---
+
+Backstage is always improving, so it's a good idea to stay in sync with the
+latest releases. Backstage is more of a library than an application or service;
+similar to `create-react-app`, the `@backstage/create-app` tool gives you a
+starting point that's meant to be evolved.
+
+## Updating Backstage versions with backstage-cli
+
+The Backstage CLI has a command to bump all `@backstage` packages you're using
+to the latest versions:
+[versions:bump](https://backstage.io/docs/cli/commands#versionsbump).
+
+```bash
+npx backstage-cli versions:bump
```

Comment:
This should be either
```
npx @backstage/cli versions:bump
```

or better, Run
```
yarn backstage-cli versions:bump
```
at the root of your repository.

Reply:
I vote the latter. When inside the repo you want to run the repo-local one explicitly.